packages/cozy-stack-client/src/DocumentCollection.js

```diff
@@ -76,16 +96,12 @@ class DocumentCollection {
   /**
    * `normalizeDoctype` for api end points returning json api responses
    *
-   * @private
    * @param {string} doctype - Document doctype
    * @returns {Function} (data, response) => normalizedDocument
    *                                        using `normalizeDoc`
    */
   static normalizeDoctypeJsonApi(doctype) {
-    return function(data, response) {
-      // use the "data" attribute of the response
-      return normalizeDoc(data, doctype)
-    }
+    return normalizeDoctypeJsonApi(doctype)
   }
```

Comment:
nit: seems like we can remove this entire method and replace all calls by the module whide `normalizeDoctypeJsonApi()`